### PR TITLE
docs: revert docs

### PR DESCRIPTION
**Pull request by yoyo837**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

File: components/timeline/index.en-US.md [hunk 1]
```diff
@@ -28,15 +28,15 @@ Vertical display timeline.
 
 ```__react
 import Alert from '../alert';
-ReactDOM.render(<Alert message="After version 5.3.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
+ReactDOM.render(<Alert message="After version 5.2.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
```
Comment (li-jia-nan) [inline components/timeline/index.en-US.md:31]: 应该是 <Alert message="xxxx"></Alert> 这种写法，不需要 ReactDOM.render 了

File: components/timeline/index.zh-CN.md [hunk 2]
```diff
@@ -29,15 +29,15 @@ demo:
 
 ```__react
 import Alert from '../alert';
-ReactDOM.render(<Alert message="After version 5.3.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
+ReactDOM.render(<Alert message="After version 5.2.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
```
Comment (li-jia-nan) [inline components/timeline/index.zh-CN.md:32]: 同上

File: components/timeline/index.en-US.md (hunk 1 above)
Comment (yoyo837) [reply components/timeline/index.en-US.md:31]: 最终需要挪到 <code> 里去不?

Comment (li-jia-nan) [reply components/timeline/index.en-US.md:31]: 不需要

Comment (li-jia-nan) [reply components/timeline/index.en-US.md]: 1. `import Alert from '../alert';` 也不需要了
2. 不能是单标签，必须是 `<Alert></Alert>`
3. message 里面的尖括号需要转义，用 `&lt;` 和 `&gt;` 代替

File: components/timeline/index.zh-CN.md [hunk 3]
```diff
@@ -27,17 +27,14 @@ demo:
 <code src="./demo/label.tsx">标签</code>
 <code src="./demo/wireframe.tsx" debug>线框风格</code>
 
-```__react
-import Alert from '../alert';
-ReactDOM.render(<Alert message="After version 5.3.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
-```
+<Alert message="After version 5.2.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />
```
Comment (li-jia-nan) [inline components/timeline/index.zh-CN.md:30]: 这里应该是中文吧？

File: components/timeline/index.en-US.md [hunk 4]
```diff
@@ -26,17 +26,14 @@ Vertical display timeline.
 <code src="./demo/label.tsx">Label</code>
 <code src="./demo/wireframe.tsx" debug>Wireframe</code>
 
-```__react
-import Alert from '../alert';
-ReactDOM.render(<Alert message="After version 5.3.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
-```
+<Alert message="After version 5.2.0, we provide a simpler usage &ltTimeline items={[...]} /&gt;  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0."></Alert>
```
Comment (li-jia-nan) [inline components/timeline/index.en-US.md:29]:
```suggestion
<Alert message="After version 5.2.0, we provide a simpler usage &lt;Timeline items={[...]} /&gt;  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0."></Alert>
```

File: components/timeline/index.zh-CN.md [hunk 5]
```diff
@@ -27,17 +27,14 @@ demo:
 <code src="./demo/label.tsx">标签</code>
 <code src="./demo/wireframe.tsx" debug>线框风格</code>
 
-```__react
-import Alert from '../alert';
-ReactDOM.render(<Alert message="After version 5.3.0, we provide a simpler usage <Timeline items={[...]} />  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />, mountNode);
-```
+<Alert message="After version 5.2.0, we provide a simpler usage &lt;Timeline items={[...]} /&gt;  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0." />
```
Comment (li-jia-nan) [inline components/timeline/index.zh-CN.md:30]:
```suggestion
<Alert message="After version 5.2.0, we provide a simpler usage &lt;Timeline items={[...]} /&gt;  with better performance and potential of writing simpler code style in your applications. Meanwhile, we deprecated the old usage in browser console, we will remove it in antd 6.0."></Alert>
```

**Review by li-jia-nan — APPROVED** [review]

**Comment by li-jia-nan** [comment]
image
image

**Pull request merged**